### ISSUE TITLE
problems with centos 8.3 > rhel 8.3 upgrade

### ISSUE THREAD

**Issue by T5wxnxWgbJF6** [oamg/convert2rhel#143]
I'm not able to upgrade from centos 8.3, seems like the issue is with the centos-release package which was renamed to centos-linux-release in 8.3. it doesn't get removed anymore and then fails to install redhat-release because of yum transaction errors.
convert2rhel completes anyways but system can't boot. trying to create grub.cfg manually shows the problem:

```
grub2-mkconfig -o /boot/efi/EFI/redhat/grub.cfg
sed: can't read /etc/system-release: No such file or directory
```

Am I doing something wrong or is 8.3 just not possible yet?
Thanks!

**Comment by bocekm** [comment]
Hi @T5wxnxWgbJF6, thanks for reaching out. We've learned about it couple of days ago. There's a fix for that already: #140.
You can updating the config file manually before running the convert2rhel (located at /usr/share/convert2rhel/configs/centos-8-x86_64.cfg) based on the fix, or wait on a new convert2rhel version 0.15 we'll be releasing soon.

**Comment by T5wxnxWgbJF6** [comment]
Great, that worked, thanks!
